### DISCUSSION TITLE
请教一个TABLE data-key值问题。

### DISCUSSION THREAD

**Discussion by JinyChen** [layui/layui#1797] [问答] [answered]
在单页面开发中，每次刷新TABLE的data-key都会增加1，请问如何保持data-key="1-0-0"呢？
![2024-04-17_112104](https://github.com/layui/layui/assets/35716328/2aded876-e665-4f90-b6f1-9d1ad0f40d97)

**Comment by Sight-wcg** [comment, marked as answer]
这是预期的行为，这个 key 是内部实例的索引。可以通过下面的方法获取到当前表格实例的索引。

```
var tbInst = table.render({
  done:functon(){
    var index = this.index
  }
})
var index2 = tbInst.config.index 
```
